### ISSUE TITLE
plot_type:boxplot  not supported in custom content ?

### ISSUE THREAD

**Issue by lindenb** [MultiQC/MultiQC#2689]
### Description of bug

unless I'm wrong, boxplots are now available ( https://multiqc.info/docs/development/plots/#box-plots ) but they are not available in custom content ?

### File that triggers the error

{
    "id": "section_idef5277c9970edf7c71b82b18db2775a3",
    "section_name": "name",
    "description": "desc",
    "plot_type": "box.plot",
    "pconfig": {
        "id": "plot__idef5277c9970edf7c71b82b18db2775a3",
        "title": "title ",
        "xlab": "depth",
        "ylab": "collection"
    },
    "data": {
        "X0 (N=1)": [
            26.44
        ],
        "XX (N=18)": [
            21.13,
            22.29,
            25.64,
            26.77,
            28.07,
            28.82,
            31.62,
            36.05,
            36.28,
            43.52,
            43.74,
            44.46,
            44.52,
            45.02,
            52.47,
            52.83,
            56.29,
            60.46
        ],
        "XY (N=4)": [
            15.24,
            21.75,
            31.11,
            45.39
        ]
    }
}

### MultiQC Error log

```console
I tried the file above with 

 "plot_type": "box"
 "plot_type": "boxplot"
 "plot_type": "box.plot"

but got:



$ multiqc --force jeter_mqc.json 

/// MultiQC 🔍 v1.22.3

       file_search | Search path: /home/lindenbaum-p/jeter_mqc.json
         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1/1  
    custom_content | Error - custom content plot type 'plot' not recognised for content ID section_idef5277c9970edf7c71b82b18db2775a3
    custom_content | section_idef5277c9970edf7c71b82b18db2775a3: Found 1 samples (plot)
     write_results | Data        : multiqc_data   (overwritten)
     write_results | Report      : multiqc_report.html   (overwritten)
           multiqc | MultiQC complete


$ multiqc --force jeter_mqc.json 

/// MultiQC 🔍 v1.22.3

       file_search | Search path: /home/lindenbaum-p/jeter_mqc.json
         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1/1  
    custom_content | Error - custom content plot type 'boxplot' not recognised for content ID section_idef5277c9970edf7c71b82b18db2775a3
    custom_content | section_idef5277c9970edf7c71b82b18db2775a3: Found 1 samples (boxplot)
     write_results | Data        : multiqc_data   (overwritten)
     write_results | Report      : multiqc_report.html   (overwritten)
           multiqc | MultiQC complete

$ multiqc --force jeter_mqc.json 

/// MultiQC 🔍 v1.22.3

       file_search | Search path: /home/lindenbaum-p/jeter_mqc.json
         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1/1  
    custom_content | Error - custom content plot type 'box.plot' not recognised for content ID section_idef5277c9970edf7c71b82b18db2775a3
    custom_content | section_idef5277c9970edf7c71b82b18db2775a3: Found 1 samples (box.plot)
     write_results | Data        : multiqc_data   (overwritten)
     write_results | Report      : multiqc_report.html   (overwritten)
           multiqc | MultiQC complete
```
```


### Before submitting

- [X] I have read the [troubleshooting documentation](https://multiqc.info/docs/usage/troubleshooting/).
- [X] I am using the latest release of MultiQC.
- [X] I have included a full MultiQC log, not truncated.
- [X] I have attached an input file (**.zip** if necessary) that triggers the error.

**Comment by vladsavelyev** [comment]
Sorry about this issue! `"plot_type": "box"` is now supported in the latest codebase, which will be released as `1.23` today.

**Comment by lindenb** [comment]
THANKS !!
